lib/queue.rb

```diff
@@ -1,31 +1,64 @@
 class Queue
-
+  
   def initialize
-    # @store = ...
-    raise NotImplementedError, "Not yet implemented"
+    @store = Array.new(10)
+    @front = 0
+    @back = 0
   end
-
+  
   def enqueue(element)
```

Comment:
This method could use being broken into some helper methods.  Otherwise it seems to work.